src/histolab/slide.py

```diff
@@ -135,6 +145,59 @@ def level_dimensions(self, level: int = 0) -> Tuple[int, int]:
                 f"{len(self._wsi.level_dimensions)}"
             )
 
+    def level_magnification_factor(self, level: int = 0) -> str:
+        """Return the magnification factor at the specified level.
+
+         Notice that the conversion level-magnification can be computed only
+        if the native magnification is available in the slide metadata.
+
+        Parameters
+        ---------
+        level : int
+            The level which magnification factor is requested, default is 0.
+
+        Returns
+        -------
+        magnification factor : str
+            Magnification factor at speficied level
+
+        Raises
+        ------
+        LevelError
+            If the specified level is not available.
+        SlidePropertyError
+            If the slide's native magnification is not available in the file
+            metadata.
```

Comment:
I think here we should add an `or` in the sense that we raise that execption in 2 cases and we mentioned only 1. So I would include 
```
or if Downsample factor for given level is not available.
```